tests/unit/utils/global-text-region-to-model-range-test.ts

```diff
@@ -0,0 +1,84 @@
+import { module, test } from "qunit";
+import ModelPosition from "@lblod/ember-rdfa-editor/model/model-position";
+import ModelRange from "@lblod/ember-rdfa-editor/model/model-range";
+import {vdom} from "@lblod/ember-rdfa-editor/model/util/xml-utils";
+import ModelElement from "@lblod/ember-rdfa-editor/model/model-element";
+import globalTextRegionToModelRange from "@lblod/ember-rdfa-editor/utils/global-text-region-to-model-range";
+
+module("Unit | Utility | global-offset-to-path", function () {
+  test("returns the correct range for a simple text", function (assert) {
+    // language=XML
+    const {root, textNodes: {testNode}} = vdom`<div><text __id="testNode">abc</text></div>`;
+    const calculatedRange = globalTextRegionToModelRange(root as ModelElement, 0, 3);
+    const range = ModelRange.fromInTextNode(testNode,0, 3);
+    assert.deepEqual(range, calculatedRange);
```

Comment:
For ranges there is also `sameAs` 
I dont think it matters really which one we use but just so you know